lib/danger/commands/local.rb

```diff
@@ -9,16 +10,37 @@ class Local < Runner
     self.command = "local"
 
     def initialize(argv)
-      @dangerfile_path = "Dangerfile" if File.exist? "Dangerfile"
```

Comment:
This is duplicate logic from https://github.com/danger/danger/blob/master/lib/danger/commands/runner.rb#L29